src/common/prometheus/__init__.py

```diff
@@ -1,3 +1,16 @@
-from common.prometheus.utils import Histogram
+from typing import Any
 
-__all__ = ("Histogram",)
+_utils = ("Histogram",)
+
+
+def __getattr__(name: str) -> Any:
+    """
+    Since utils imports django.conf.settings, we lazy load any objects that
+    we want to import to prevent django.core.exceptions.ImproperlyConfigured
+    due to settings not being configured.
+    """
+    if name in _utils:
+        from common.prometheus import utils
+
+        return getattr(utils, name)
+    raise AttributeError(name)
```

Comment:
This is pretty ugly, but might actually save us pain in the future, as well as fixing the pain I experienced. There are a couple of alternatives: 

1. In `utils.py`, use a factory instead of subclassing `Histogram`, and perform a local import of `django.conf.settings`
2. Take a slightly different approach and just split up the logical pieces of `prepare_prom_multiproc_dir` into generic functions where necessary like `clear_sub_dirs(dir: str)`, and just call it step by step in `ensure_cli_env`